src/MemProf.f90

```diff
@@ -163,12 +163,12 @@ SUBROUTINE edit_MemProf(thisMP,name)
           thisMP%print_banner=.FALSE.
         ENDIF
         CALL thisMP%pe%world%barrier()
-        CALL getProcMemInfo(tmpL1,tmpL2)
-        thisMP%mem_current=REAL(tmpL1,SRK)/(1024.0_SRK*1024.0_SRK)
+        CALL getProcMemInfo(tmpL1,tmpL2) !tmpL1 in kB, and tmpL2 in bytes
+        thisMP%mem_current=REAL(tmpL1,SRK)*1000.0_SRK/(1024.0_SRK*1024.0_SRK*1024.0_SRK)
```

Comment:
why is there a factor of 1000/1024?

Reply:
See the comments in the commit message.
kB = 1000 bytes. I'm converting to KB.